LibVLCSharp/LibVLCSharp.csproj

```diff
@@ -1,4 +1,5 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras/1.6.46">
+﻿<!--Project Sdk="MSBuild.Sdk.Extras/1.6.46"-->
```

Comment:
You should put this back as it was.

Reply:
Good catch, I made this change to allow dotnet-format tool to run without errors (It does not seem to support custom build SDK yet)